### PR TITLE
[org] Add org-cliplink package

### PR DESCRIPTION
**Pull request by bb2020**
`org-cliplink` is amazing. Inserts link from kill ring into `org` buffer with the page's title.

Edit: I just selected an arbitrary binding. Maybe `SPC m i L` would be a better choice.

### CODE REVIEW

File: layers/+emacs/org/README.org [hunk 1]
```diff
@@ -501,9 +501,10 @@ are also available.
 
 ** Links
 
-| Key binding | Description       |
-|-------------+-------------------|
-| ~SPC m x o~ | org-open-at-point |
+| Key binding | Description                                |
+|-------------+--------------------------------------------|
+| ~SPC m x o~ | org-open-at-point                          |
+| ~SPC m i c~ | org-cliplink (insert link with page title) |
```
Comment (smile13241324) [inline layers/+emacs/org/README.org]: Please add the binding under `Element insertion` right under `SPC m i l`

File: layers/+emacs/org/packages.el [hunk 2]
```diff
@@ -645,6 +646,13 @@ Headline^^            Visit entry^^               Filter^^                    Da
       (add-hook 'org-present-mode-hook 'spacemacs//org-present-start)
       (add-hook 'org-present-mode-quit-hook 'spacemacs//org-present-end))))
 
+(defun org/init-org-cliplink ()
+  (use-package org-cliplink
+    :defer t
+    :init
+    (spacemacs/set-leader-keys-for-major-mode 'org-mode
+      "ic" 'org-cliplink)))
```
Comment (smile13241324) [inline layers/+emacs/org/packages.el]: Please change the binding to `SPC m i L` instead.

**Review by smile13241324 — CHANGES_REQUESTED** [review]
Hi @bb2020,

I have just reviewed your PR, and would like to request some changes.
Please check my comments and mention me when you are ready for another review.

**Comment by bb2020** [comment]
@smile13241324 Ready

**Review by smile13241324 — APPROVED** [review]

**Comment by smile13241324** [comment]
Thank you for contributing to Spacemacs!
The PR has been cherry-picked into the develop branch, you can safely delete your branch.

**Pull request closed without merging**